occurrences/tests/test_api.py

```diff
@@ -1463,68 +1468,8 @@ def test_enrol_full_people_count_seat_type_occurrence(
         enrolment_end_days=2,
         auto_acceptance=auto_acceptance,
     )
-    occurrence = OccurrenceFactory(
-        start_time=datetime(2020, 1, 6, 0, 0, 0, tzinfo=timezone.now().tzinfo),
-        p_event=p_event_1,
-        min_group_size=10,
-        max_group_size=20,
-        amount_of_seats=34,
-    )
 
-    # After 20 people there are 14 seats left
-    EnrolmentFactory(occurrence=occurrence, study_group=study_group_20)
-    # Approve the enrolment to reduce the remaining seat
-    # NOTE: currently also the pending enrolments takes seats
-    # Enrolment.objects.first().approve()
-
-    # A group of 15 woul make the event over booked by 1!
-    variables = {
-        "input": {
-            "occurrenceIds": [to_global_id("OccurrenceNode", occurrence.id)],
-            "studyGroup": {
-                "person": {
-                    "id": to_global_id("PersonNode", study_group_15.person.id),
-                    "name": study_group_15.person.name,
-                    "emailAddress": study_group_15.person.email_address,
-                },
-                "unitName": "To be created group",
-                "groupSize": study_group_15.group_size,
-                "groupName": study_group_15.group_name,
-                "studyLevels": [sl.upper() for sl in study_group_15.study_levels.all()],
-                "amountOfAdult": study_group_15.amount_of_adult,
-            },
-        }
-    }
-    executed = api_client.execute(ENROL_OCCURRENCE_MUTATION, variables=variables)
-    assert_match_error_code(executed, NOT_ENOUGH_CAPACITY_ERROR)
-    assert mock_send_sms.call_count == 0
-    assert len(mail.outbox) == 0
-
-
-# The auto_acceptance calls Enrolment.approve,
-# so it needs to be tested with both the boolean values.
-@pytest.mark.parametrize("auto_acceptance", [True, False])
-@patch("occurrences.services.notification_service.send_sms")
-def test_multi_enrol_full_people_count_seat_type_occurrence(
-    mock_send_sms,
-    auto_acceptance,
-    api_client,
-    mock_update_event_data,
-    mock_get_event_data,
-    occurrence,
-    notification_template_enrolment_approved_en,
-    notification_template_enrolment_approved_fi,
-    notification_sms_template_enrolment_approved_en,
-    notification_sms_template_enrolment_approved_fi,
-):
-    study_group_15 = StudyGroupFactory(group_size=15)
-    study_group_20 = StudyGroupFactory(group_size=20)
-    # Current date froze on 2020-01-04:
-    p_event_1 = PalvelutarjotinEventFactory(
-        enrolment_start=datetime(2020, 1, 3, 0, 0, 0, tzinfo=timezone.now().tzinfo),
-        enrolment_end_days=2,
-        auto_acceptance=auto_acceptance,
-    )
+    # if is_multi_enrolment is True, multiple occurrences are needed
     valid_occurrence = OccurrenceFactory(
```

Comment:
If you want to do purely refactoring here (i.e. not changing functionality) then adding a
```python
if is_multi_enrolment:
    valid_occurence = ...
```
would make the code work more like before (i.e. not creating another Occurence when is_multi_enrolment is False).

Reply:
had it like that, but then it would have been Outbound. Did a small refactor here now.

Reply:
> had it like that, but then it would have been Outbound. Did a small refactor here now.

What do you mean by Outbound? Wanted to be sure and checked it locally that the test passes with the added if clause.